index.html

```diff
@@ -109,11 +111,84 @@
+                                  <polygon id="Path" points="29.2179487 8.78378378 32.6153846 8.78378378 32.6153846 10.8108108 29.2179487 10.8108108"></polygon>
+                                  <path d="M25.7260004,8.75355875 L27.1794872,8.75355875 L27.1794872,6.75675676 L25.3465186,6.75675676 C25.1035249,6.75675676 24.8703764,6.86205686 24.6985356,7.04925705 L23.7820513,8.04765805 L25.0780173,9.45945946 L25.7260004,8.75355875 Z" id="Path"></path>
+                                  <path d="M36.2069597,28.3783784 L46.6904762,28.3783784 C47.172933,28.3783784 47.5641026,27.9749965 47.5641026,27.4774775 L47.5641026,19.3693694 C47.5611165,16.8826542 45.6074013,14.8679441 43.1959707,14.8648649 L39.7014652,14.8648649 C37.2900346,14.8679441 35.3363194,16.8826542 35.3333333,19.3693694 L35.3333333,27.4774775 C35.3333333,27.9749965 35.7245029,28.3783784 36.2069597,28.3783784 Z M45.8168498,19.3693694 L45.8168498,26.5765766 L42.3223443,26.5765766 L42.3223443,20.4827386 L44.0862335,16.837785 C45.1211053,17.2231313 45.8125841,18.2344454 45.8168498,19.3693694 Z M42.2157005,16.6666667 L41.4487179,18.2520411 L40.6817354,16.6666667 L42.2157005,16.6666667 Z M37.0805861,19.3693694 C37.0848518,18.2344454 37.7763305,17.2231313 38.8112024,16.837785 L40.5750916,20.4827386 L40.5750916,26.5765766 L37.0805861,26.5765766 L37.0805861,19.3693694 Z" id="Shape" fill-rule="nonzero"></path>
+                                  <path d="M45.9786325,10.4469854 L45.9786325,9.56340956 C46.478958,9.56340956 46.8846154,9.16778504 46.8846154,8.67983368 C46.8846154,5.75212575 44.450671,3.37837838 41.4487179,3.37837838 C38.4467649,3.37837838 36.0128205,5.75212575 36.0128205,8.67983368 C36.0128205,9.16778504 36.4184779,9.56340956 36.9188034,9.56340956 L36.9188034,10.4469854 C36.9188034,12.8867421 38.9470903,14.8648649 41.4487179,14.8648649 C43.9503456,14.8648649 45.9786325,12.8867421 45.9786325,10.4469854 Z M41.4487179,5.14553015 C42.9996828,5.14768729 44.3776793,6.10978413 44.8797743,7.54084914 C44.1843616,7.22805986 43.5469001,6.80439207 42.9943744,6.28710334 C42.6404748,5.94281932 42.0680422,5.94281932 41.7141426,6.28710334 C40.5555639,6.88722731 39.309395,7.30873795 38.0185464,7.53739767 C38.5215261,6.10805839 39.8990802,5.14725588 41.4487179,5.14553015 Z M41.4487179,13.0977131 C39.9477414,13.0977131 38.7307692,11.9108394 38.7307692,10.4469854 L38.7307692,9.18978818 C39.9503956,8.92531933 41.1372863,8.5340092 42.2715345,8.02232897 C42.847506,8.48439432 43.4849676,8.8675073 44.1666667,9.16217643 L44.1666667,10.4469854 C44.1666667,11.9108394 42.9496945,13.0977131 41.4487179,13.0977131 Z" id="Shape" fill-rule="nonzero"></path>
+                              </g>
+                          </g>
+                      </g>
+                  </g>
+              </svg>
+            </div>
+            <div class="pe-cta-text">
+              <a href="https://youtube.perfects.engineering">YouTube Channel</a>
+              <p>Helpful tutorials on software engineering 
```

Comment:
Please change this text to 'Watch videos on engineering Android & Blockchain software' or something like that.